homeassistant/components/zha/const.py

```diff
@@ -0,0 +1,84 @@
+"""Constants for the ZHA integration."""
+
+DOMAIN = "zha"
+EZSP_OVERWRITE_EUI64 = (
+    "i_understand_i_can_update_eui64_only_once_and_i_still_want_to_do_it"
+)
+CONF_USE_THREAD = "use_thread"
+
+ATTR_ACTIVE_COORDINATOR = "active_coordinator"
+ATTR_AVAILABLE = "available"
+ATTR_DEVICE_TYPE = "device_type"
+ATTR_ENDPOINT_NAMES = "endpoint_names"
+ATTR_IEEE = "ieee"
+ATTR_LAST_SEEN = "last_seen"
+ATTR_LQI = "lqi"
+ATTR_MANUFACTURER = "manufacturer"
+ATTR_MANUFACTURER_CODE = "manufacturer_code"
+ATTR_MODEL = "model"
```

Comment:
Is also present in `homeassistant.const`

Reply:
fixed